spec/lib/supplier_import_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require './lib/supplier_import'
+require './lib/supplier_import_row'
+
+RSpec.describe SupplierImport do
+  describe '#initialize' do
+    it 'accepts CSV data as a string input' do
+      csv = "frameworkreference,suppliername\nRM1234,Cheese & Crackers LLP\nRM5678,Vaughan Legal\n"
+
+      expect { SupplierImport.new(csv) }.to_not raise_error
+    end
+  end
+
+  describe '#run!' do
+    it 'calls SupplierImportRow.import! for each row contained in the CSV' do
+      # TODO: This should probably stub calls to SupplierImportRow
```

Comment:
I think it's fine not to stub anything here. I prefer not to in fact as this spec then serves as an integration for the two dependent classes and is less susceptible to brittleness if the row class or the way it's used should change. Lose the comment I reckon to avoid it becoming a loose thread for future developers to ponder on.